src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/DomainEventEntity.kt

```diff
@@ -108,7 +110,7 @@ data class DomainEventEntity(
   val occurredAt: OffsetDateTime,
   val createdAt: OffsetDateTime,
   @Type(type = "com.vladmihalcea.hibernate.type.json.JsonType")
-  val data: String,
+  var data: String,
```

Comment:
you can use the existing DomainEventRepository.updateData() function, so no need to make this mutable